persistence/src/main/resources/demoInstitutions.csv

```diff
@@ -1,4 +1,4 @@
-2015|0|3|2|1|1234|123|1234|1234|1234|test@test.com|test@test.com|test@test.com|Bank 0|VA|Test|1|1|123|123||||1|100|1|||||
-2015|1|2|2|0|4321|123|1234|1234|1234|test@test.com|test@test.com|test@test.com|Bank 1|VA|Test|1|1|123|123||||1|100|1|||||
-2015|2|3|2|1|1234|123|1234|1234|1234|test@test.com|test@test.com|test@test.com|Bank 2|VA|Test|1|1|123|123||||1|100|1|||||
-2015|4|3|2|1|1234|123|1234|1234|1234|test@test.com|test@test.com|test@test.com|Bank 3|VA|Test|1|1|123|123||||1|100|1|||||
+2015|0|3|2|1|1234|123|1234|1234|1234|test@bank0.com|test@bank0.com|test@bank0.com|Bank 0|VA|Test|1|1|123|123||||1|100|1|||||
```

Comment:
It's probably worth having a test where a given institution has different email domains, that way you can test that all unique values are returned in the `domains` attribute, and that the same institution can be found when searching for each of its domains.